presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaTableHandle.java

```diff
@@ -52,6 +53,8 @@
     private final String messageDataFormat;
     private final Optional<String> keyDataSchemaLocation;
     private final Optional<String> messageDataSchemaLocation;
+    private final Optional<Map<String, String>> keyDecoderParams;
```

Comment:
Can we have it as Map<String, String> it can be empty if the table doesn't add any new parameter